python/tests/orderby_tests/sqlite_runner.py

```diff
@@ -0,0 +1,89 @@
+import sys
+import sqlite3
+from types import ModuleType
+import re
+
+from tests.aggregate_tests.aggtst_base import Table, View, TstAccumulator, DEBUG
+from tests.aggregate_tests.atest_register_tests import register_tests_in_module
+
+
+class SQLiteRunner:
+    """Run Tables and Views on SQLite using a sqlite3 cursor"""
+
+    def __init__(self, con: sqlite3.Connection):
+        self.con = con
+        self.cur = con.cursor()
+
+    def run_table(self, table: Table):
+        self.cur.execute(table.get_sql())
+        for row in table.get_data():
+            values = list(row["insert"].values())
+            placeholders = ", ".join(["?"] * len(values))
+            sql = f"INSERT INTO {table.name} VALUES ({placeholders})"
+            self.cur.execute(sql, values)
```

Comment:
what happens on error? Does the program terminate?
Or do you have to check the result?

Reply:
what makes that test be ignored?